db/seeds.d/100-installation_media.rb

```diff
@@ -6,17 +6,7 @@
 Medium.without_auditing do
   [
     {
-      :name => "CentOS 7 mirror",
-      :os_family => "Redhat",
-      :path => "http://mirror.centos.org/centos/$major/os/$arch",
-    },
-    {
-      :name => "CentOS Stream",
-      :os_family => "Redhat",
-      :path => "http://mirror.centos.org/centos/$major-stream/BaseOS/$arch/os",
-    },
-    {
-      :name => "CentOS Stream 9 mirror",
+      :name => "CentOS mirror",
```

Comment:
I think this may need a DB migration to rename the existing entry.

Reply:
Yup, this will definitely require a migration.

On fresh dev machine:
```
[2] pry(main)> Medium.where("name like '%9%'").all
=> [#<Medium:0x00007f08ff4f6aa0
  id: 3,
  name: "CentOS Stream 9 mirror",
  path: "http://mirror.stream.centos.org/$major-stream/BaseOS/$arch/os",
  created_at: Tue, 30 Jul 2024 10:43:46.058948000 UTC +00:00,
  updated_at: Tue, 30 Jul 2024 10:43:46.058948000 UTC +00:00,
  media_path: nil,
  config_path: nil,
  image_path: nil,
  os_family: "Redhat">]
```

Reply:
Should we also delete the Centos Stream mirror and use that in place of the Centos one we are switching to? I see Stream 9 and Stream are both in there?
```
2	CentOS Stream	http://mirror.centos.org/centos/$major-stream/BaseOS/$arch/os	2024-07-19 15:57:37.289
3	CentOS Stream 9 mirror	http://mirror.stream.centos.org/$major-stream/BaseOS/$arch/os	2024-07-19 15:57:37.314
```

Reply:
Deletion is something we usually don't do. Users may have modified them to point to some internal link. That's a can of worms we prefer to keep closed